addon/webextension/background.js

```diff
@@ -8,10 +8,50 @@
 
 
 // constants used by particular effects
-const WORDS = "dark army disintegration data privacy internet delete".split(/\s+/);
-const XHEADERSITES = ['<all_urls>'];
-const XHEADERNAME = 'dontdeleteme';
-const XHEADERVALUE = '1057'
+const WORDS =
```

Comment:
I'm really surprised "data" is not on this list!

Reply:
me too!